x/oracle/types/params.go

```diff
@@ -0,0 +1,140 @@
+package types
+
+import (
+	"fmt"
+	"github.com/btcsuite/btcd/btcec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"time"
+)
+
+var (
+	KeyOraclePublicKey          = []byte("OraclePublicKey")
+	KeyOraclePubKeyRemoteReport = []byte("OraclePubKeyRemoteReport")
+	KeyUniqueID                 = []byte("UniqueID")
+	KeyVoteParams               = []byte("VoteParams")
+	KeySlashParams              = []byte("SlashParams")
+)
+
+var _ paramtypes.ParamSet = (*Params)(nil)
+
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+func DefaultParams() Params {
+	return Params{
+		OraclePublicKey:          nil,
+		OraclePubKeyRemoteReport: nil,
+		UniqueId:                 "",
+		VoteParams: VoteParams{
+			VotingPeriod: 30 * time.Second,
+			JailPeriod:   10 * time.Minute,
+			Quorum:       sdk.NewDecWithPrec(1, 3),
```

Comment:
Is `0.001` the value that you intended? If you wanted `1/3`, you can use `sdk.NewDec(1).Quo(sdk.NewDec(3))`

Reply:
Right. I meant `0.001`

Reply:
Okay. Then, if only 1 out of 1000 votes yes, the tally result would be finalized with yes (depending on your implementation).

Reply:
Ah, I misunderstood.
I'll have to set it up as you said in the beginning